.rubocop.yml

```diff
@@ -1,9 +1,7 @@
 AllCops:
-  Include:
-    - '**/Rakefile'
-    - '*.gemspec'
```

Comment:
Why is this removed?

Reply:
Because it seemed unnecessary to limit Rubocop so that it didn't run on any of the actual code files.

Reply:
Oh my… I thought it would be merged like `Exclude` does. Silly me.

But can we add these to the list of checked files too?

Reply:
Yes, you're right. My assumptions were incorrect.